src/core/router.go

```diff
@@ -111,6 +111,7 @@ func initRouters() {
 
 	beego.Router("/api/internal/syncregistry", &api.InternalAPI{}, "post:SyncRegistry")
 	beego.Router("/api/internal/renameadmin", &api.InternalAPI{}, "post:RenameAdmin")
+	beego.Router("/api/internal/configurations", &api.ConfigAPI{}, "get:GetInternalConfig")
```

Comment:
Suggest replace 'internal' to avoid set bars for the system integration.

* `/api/configs/{scope}`
  * `/api/configs/usersettings`
  * `/api/configs/systemsettings`
* `/api/configs/{scope}/{group}`
* `/api/configs/{scope}/{group}/{key}`

Reply:
* /api/configs/usersettings/emails
  * /api/configs/usersettings/emails/url
  * /api/configs/usersettings/emails/cc